### DISCUSSION TITLE
[Question][Environment] ERROR: Cannot connect with current user ID in SecuwaySSN

### DISCUSSION THREAD

**Discussion by hyerinshelly** [prosyslab-classroom/cs348-information-security#483] [Q&A]
I’m struggling with SecuwaySSN from yesterday. It didn’t work well, so I've tried many times and now it seems that my user ID is blocked. It keeps giving me the error "Cannot connect with current user ID (현재 사용자 ID로는 접속할 수 없습니다)". Do I need additional permission for this? Or is there any solution? 

* My user ID is CS448-30. For information, I used 'https://kcloudvpn.kaist.ac.kr' for VPN URL and right most column in spreadsheet for the password.

**Comment by RiceBiscuits** [comment]
CS448-30 ID is available now.
Important thing to login the vpn is that ID is case-sensitive.
Please let me know if you have a difficulty to solve the problem.

**Comment by hyerinshelly** [comment]
> CS448-30 ID is available now.
> Important thing to login the vpn is that ID is case-sensitive.
> Please let me know if you have a difficulty to solve the problem.

 Now it's working well. Thanks for the noticing. Thank you so much!
